src/boardlib/api/aurora.py

```diff
@@ -541,24 +591,31 @@ def calculate_tries_total(group):
     return group
 
 
-def logbook_entries(board, username, password, grade_type="font", db_path=None):
-    login_info = login(board, username, password)
-    token = login_info["token"]
-    user_id = login_info["user_id"]
-    
-    bids_entries = list(bids_logbook_entries(board, username, password, db_path))
+def logbook_entries(board, username, password=None, token=None, user_id=None, grade_type="font", db_path=None, aggregate=False):
```

Comment:
We don't need a function to support both forms of arguments. We should just have the command line call an extra function to fetch the token and then have `logbook_entries` require a token.